bugsnag.go

```diff
@@ -57,21 +88,32 @@ func AutoNotify(rawData ...interface{}) {
 		severity := defaultNotifier.getDefaultSeverity(rawData, SeverityError)
 		state := HandledState{SeverityReasonHandledPanic, severity, true, ""}
 		rawData = append([]interface{}{state}, rawData...)
-		defaultNotifier.NotifySync(errors.New(err, 2), true, rawData...)
+		defaultNotifier.NotifySync(append(rawData, errors.New(err, 2), true)...)
 		panic(err)
 	}
 }
 
 // Recover logs a panic on a goroutine and then recovers.
+// Although it's not strictly enforced, it's highly recommended to pass a
+// context.Context object that has at one-point been returned from
+// bugsnag.StartSession. Doing so ensures your stability score remains accurate,
```

Comment:
In this case, the stability score is unaffected because Recover will report crashes as handled and won't contribute to the stability score. We may need to provide another recover method or change the example to include how to set it up to affect the stability score.